ros2/Dockerfile

```diff
@@ -112,10 +111,18 @@ RUN . /opt/ros/${ROS_DISTRO}/setup.sh \
       --ignore-src \
       --skip-keys="python-rosdep python-catkin-pkg python-rospkg" \
       --rosdistro="${ROS_DISTRO}" \
- && (test -f /.dockerinstall/prebuild.sh \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# We copy pkgs.rosinstall again because copy needs at least one file that exists
+# But we explicitly copy prebuild.sh and files that are used by it in pbfiles
+COPY "${DIRECTORY}"/pkgs.rosinstall  "${DIRECTORY}"/pbfiles* "${DIRECTORY}"/prebuild.sh* /.dockerinstall/
+RUN apt-get update\
+     && (test -f /.dockerinstall/prebuild.sh \
      && (echo "running prebuild step..." && /bin/bash /.dockerinstall/prebuild.sh || exit 1) \
```

Comment:
Indent 122 and 123 to indicate nested conditionals.

Reply:
Done